requirements.tex

```diff
@@ -28,25 +28,25 @@ \subsection{Batch System} \label{sec:offlineprod}
 The batch system and associated interactive nodes are also used extensively by staff for development of future versions of the LSST Science Pipelines code.\cite{2019ASPC..523..521B}
 Though most science user access will be via the Data Access Center (DAC) (see \autoref{sec:usdac})
 there are requirements for science users to have some access to batch-type processing for larger-scale, non-interactive computations on the data.\cite{DMTN-223}
-This will be controlled by a Resource Allocation Committee
+This will be controlled by a Resource Allocation Committee.
 
 \subsection{Data transfer and preservation} \label{req:dbb}
 The USDF is responsible for operating the systems that track all raw data and released data products, including managing their movement, backup, and lifetime.
 These systems must be fault-tolerant and able to catch up after failures.
```

Comment:
I wasn't sure if you meant to say the fault-tolerant systems would catch up or that you were referring to "catch-up" mode to transfer files when there are network outages... or both

Reply:
reworded slightly